src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -498,6 +498,16 @@ CustomTextLayout::CustomTextLayout(IDWriteFactory2* const factory,
             glyphRunDescription.stringLength = run.textLength;
             glyphRunDescription.textPosition = run.textStart;
 
+            // Calculate the position *after* drawing, this will be used if we're in RTL.
+            // Original comment:
+            // Shift origin to the right for the next run based on the amount of space consumed.
+            auto postOriginX = std::accumulate(_glyphAdvances.begin() + run.glyphStart,
+                                               _glyphAdvances.begin() + run.glyphStart + run.glyphCount,
+                                               mutableOrigin.x);
+
+            if (WI_IsFlagSet(glyphRun.bidiLevel, 1))

```

Comment:
This is probably where your style-enforcer check is failing (for not including braces on the if even though it's just one line)